web/forms.py

```
@@ -553,6 +554,50 @@ def clean(self):
         return cleaned_data
 
 
+class QuizQuestionSpecializedForm(forms.ModelForm):
+    """Form for creating specialized quiz questions."""
+
+    class Meta:
+        model = QuizQuestion
+        fields = ["text", "question_type", "explanation", "points", "image"]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)